psi4/src/psi4/libcubeprop/csg.cc

```diff
@@ -649,6 +649,33 @@ void CubicScalarGrid::compute_orbitals(std::shared_ptr<Matrix> C, const std::vec
     }
     free_block(v);
 }
+void CubicScalarGrid::compute_difference(std::shared_ptr<Matrix> C, const std::vector<int>& indices,
+                                         const std::string& label, bool square, const std::string& type) {
+    auto C2 = std::make_shared<Matrix>(primary_->nbf(), indices.size());
+    double** Cp = C->pointer();
+    double** C2p = C2->pointer();
```

Comment:
Check that `C->nirrep() == 1`, so you do not get odd failures for matrices with symmetry.

Reply:
I took the `compute_orbitals` above as a guide. Is this symmetry check necessary, and if yes, shouldn't it be done for the orbitals as well?